mmcv/ops/prroi_pool.py

```diff
@@ -65,14 +70,26 @@ def backward(
         grad_input = grad_output.new_zeros(*features.shape)
         grad_coor = grad_output.new_zeros(*rois.shape)
 
-        if features.requires_grad:
+        if features.requires_grad or TORCH_VERSION == 'parrots':
```

Comment:
Why does parrots need to do backward propagation when features do not require gradient?

Reply:
for parrots, the requires_grad attr will be optimized while constructing compute graph. so for forward or backward, this is not suitable for judgement as it will always be false.

Reply:
Okay